### DISCUSSION TITLE
KOPS 1.19 version support

### DISCUSSION THREAD

**Discussion by artbaghdasaryan** [eddycharly/terraform-provider-kops#152] [Q&A] [answered]
Hi,
When is planned to release the `KOPS 1.19` version with `addons` support ?

**Comment by eddycharly** [comment, marked as answer]
Hello @artbaghdasaryan,

I have a PR opened for this, I haven't had time to test it yet but I should be able to test and release it (at least one first alpha version) next week.

**Reply by artbaghdasaryan** [reply]
Thanks for reply

Actually there is an issue while using Your developed provider,
While creating the cluster for the first time and choosing AWS Route 53 private zone the `terraform apply` fails as `kops` is creating the `API` record in the zone and the bastion host which is in the same VPC do not resolve it ,it needs time 

Can you add a timeout for it ?
error is 
`response from 10.0.0.2()AWS internal DNS) no such host`

**Reply by eddycharly** [reply]
Thanks for reporting, can you create an issue with the details please ?

I’ll dig it to see how it can be solved.

**Reply by artbaghdasaryan** [reply]
@eddycharly did you have a chance to dig into the issue ?
As I see from the logs `kops_cluster_updater.updater` resource isn't acting properly.
It is not waiting for DNS record(api.<cluster name>) to be reachable and fails with 
`Error: Get https://api<cluster-name>/api/v1/nodes: dial tcp: lookup api.<cluster name> on 10.0.0.2:53: no such host`

The DNS zone is private and created before and assigned the VPC correctly
After 20 seconds when the DNS record is reachable the  `kops_cluster_updater.updater` is working properly

**Reply by eddycharly** [reply]
@artbaghdasaryan unfortunately i didn't have the time to look at it deeply.

From what i understand, the kops_cluster_updater should retry until the api can be resolved, i'm going to check this now.

I should have some time this week to finalize 1.19 support and test it on one of our clusters.
